src/components/GettingStarted/GettingStarted.js

```diff
@@ -3,62 +3,143 @@ import RoutePath from 'lib/routePath';
-import Page0_Overview from './Steps/Overview';
-import Page2_SimpleExample from './Steps/SimpleExample';
-
-const GettingStarted = (props) => (
-  <DocumentTitle title='Getting Started'>
-    <Breadcrumb
-      data={{
-        title: 'GETTING STARTED',
-        pathname: RoutePath.createUsablePath(
-          OrganizationsRoutes.Clusters.GettingStarted.Overview,
-          {
-            clusterId: props.match.params.clusterId,
-            orgId: props.match.params.orgId,
-          }
-        ),
-      }}
-    >
-      <div>
-        <Switch>
-          <Route
-            component={Page0_Overview}
-            exact
-            path={OrganizationsRoutes.Clusters.GettingStarted.Overview}
-          />
-          <Route
-            component={Page1_ConfigureKubeCTL}
-            exact
-            path={OrganizationsRoutes.Clusters.GettingStarted.ConfigureKubeCtl}
-          />
-          <Route
-            component={Page2_SimpleExample}
-            exact
-            path={OrganizationsRoutes.Clusters.GettingStarted.SimpleExample}
-          />
-          <Route
-            component={Page3_NextSteps}
-            exact
-            path={OrganizationsRoutes.Clusters.GettingStarted.NextSteps}
-          />
-          <Redirect
-            path='*'
-            to={OrganizationsRoutes.Clusters.GettingStarted.Overview}
-          />
-        </Switch>
-      </div>
-    </Breadcrumb>
-  </DocumentTitle>
-);
+import ConfigureKubeCTL from './Steps/ConfigureKubectl';
+import InstallIngress from './Steps/InstallIngress';
+import NextSteps from './Steps/NextSteps';
+import Overview from './Steps/Overview';
+import SimpleExample from './Steps/SimpleExample';
+
+const GettingStarted = (props) => {
+  const pathParams = {
+    orgId: props.match.params.orgId,
+    clusterId: props.match.params.clusterId,
+  };
+
+  const clusterGuideConfigurationPath = RoutePath.createUsablePath(
+    OrganizationsRoutes.Clusters.GettingStarted.ConfigureKubeCtl,
+    pathParams
+  );
+
+  const clusterGuideExamplePath = RoutePath.createUsablePath(
+    OrganizationsRoutes.Clusters.GettingStarted.SimpleExample,
+    pathParams
+  );
+
+  const clusterGuideIngressPath = RoutePath.createUsablePath(
+    OrganizationsRoutes.Clusters.GettingStarted.InstallIngress,
+    pathParams
+  );
+
+  const clusterGuideNextStepsPath = RoutePath.createUsablePath(
+    OrganizationsRoutes.Clusters.GettingStarted.NextSteps,
+    pathParams
+  );
+
+  const steps = [
+    {
+      title: 'Get Access',
+      description:
+        'Enable your Kubernetes CLI to access your Kubernetes cluster at Giant Swarm',
+      url: clusterGuideConfigurationPath,
+      routePath: OrganizationsRoutes.Clusters.GettingStarted.ConfigureKubeCtl,
+      component: ConfigureKubeCTL,
+    },
+
+    {
+      title: 'Run a simple example',
+      description:
+        'To make sure everything works as expected, let&apos;s start a hello world application',
```

Comment:
Btw, this is treated as a string when used in the component, so the `&apos;` is escaped in the output.

![image](https://user-images.githubusercontent.com/13508038/80350772-827af680-8871-11ea-8834-15915642254d.png)

Reply:
Well spotted! thanks!